pkgs/tools/admin/jetporch/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, rustPlatform,
+  fetchFromSourcehut,
+  fetchurl, openssl,
+  pkg-config,
+  writeText,
+  ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jetporch";
+  version = "unstable-2023-11-18";
```

```suggestion
  version = "unstable-2023-10-16";
```
The date has to be the commit date.
(There is also a newer commit from seven hours ago, but it doesn't look relevant for our package.)

---

Done: d66749979e721343e7c86693d4490a98d213b9f8